docs/development/README.md

```diff
@@ -11,6 +11,7 @@
 
 - Fork <https://github.com/triplea-game/triplea>
 - Clone your newly forked repository
+- Create a new branch in your fork repository and do the checkout
```

Comment:
1. I'm not sure what is meant by "do the checkout". This could be clarified.
2. This perhaps moots (1), we do have docs on how to do PRs and the typical workflow. Perhaps it would be better to add a link to those pages?
3. 
https://github.com/triplea-game/triplea/blob/master/docs/development/how-to/cli-build-commands.md

https://github.com/triplea-game/triplea/blob/master/docs/development/how-to/typical-git-workflow.md